### PR TITLE
Fix date-related and message bugs

### LINKED ISSUE #162: [PE-D] Date format issue
Help page says that d M yyyy and M yyyy are acceptable date formats. However, these date formats are not accepted by the timeline command even though its supposed to accept timeline [STARTDATE] [ENDDATE].

e.g. timeline 1 2 2020 2 2 2020  OR timeline 2 2020 3 2020

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: JeremiasLiew/ped#4

### LINKED ISSUE #164: [PE-D] Persons and events can be created with death date before birth date
Persons can be created with a death date that occurs before their birth date. This can be done by entering a death date that is earlier than the entered birth date.

This can also be done for events by entering an end date that occurs before the start date.

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: JeremiasLiew/ped#2

### LINKED ISSUE #185: Update PPP with UG and DG contributions - Lilian

### PR DESCRIPTION
**Pull request by g-lilian**

### CODE REVIEW

File: src/main/java/seedu/tp/ui/Ui.java [hunk 1]
```diff
@@ -260,7 +260,7 @@ private void listFlashcardsInPeriod(List<Flashcard> sortedFlashcards, LocalDate
     }
 
     /**
-     * Prints confirmation that study plan has been updated.
+     * Prints confirmation that study plan has been deleted.
```
Comment (Zhilin-Huang) [inline src/main/java/seedu/tp/ui/Ui.java]: Shouldn't this be kept as 'updated' since it's for the `confirmStudyPlanUpdate` method?

**Review by Zhilin-Huang — CHANGES_REQUESTED** [review]
Consider adding text Ui test for cases where start date > end date when creating event and person flashcards :)

File: src/main/java/seedu/tp/ui/Ui.java (hunk 1 above)
Comment (g-lilian) [reply src/main/java/seedu/tp/ui/Ui.java]: Oops, you're right, I meant to change the comment for the method below (deletion). Will update.

**Comment by g-lilian** [comment]
> Consider adding text Ui test for cases where start date > end date when creating event and person flashcards :)

Added!

**Review by Zhilin-Huang — APPROVED** [review]
LGTM!

**Pull request merged**